.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,10 +1,9 @@
 ---
 name: Bug report
 about: Create a report to help us improve
-title: ''
-labels: ''
-assignees: ''
-
+title: ""
+labels: ""
```

Comment:
```suggestion
labels: "bug"
```

Optional: automatically attach the `bug` label to bug reports.

Reply:
Yeah, this is great.